bin/plugin/open/selfAddIngressKey

```diff
@@ -67,14 +67,20 @@ if (not defined $pubKey) {
     my $algos    = join(' ', @algoList);
     osh_info "Please paste the SSH key you want to add. This bastion supports the following algorithms:\n";
 
+    if (grep { 'ed25519-sk' eq $_ } @algoList) {
+        osh_info "ED25519 Secure-Key: strongness[######] speed[#####], use `ssh-keygen -t ed25519-sk' to generate one";
```

Comment:
I think we should instead call the "-sk" algos "FIDO2 Ed25519" and "FIDO2 ECDSA", what do you think? FIDO2 seems to be the right keyword for people to query their preferred web search engine and get all the information they need.

Reply:
I'm fine with both naming conventions. I'd just keep the `ed25519-sk` and `ecdsa-sk` names internally as it's what `ssh-keygen -lf` outputs.